scripts/stage-docs.sh

```diff
@@ -159,7 +162,7 @@ gh repo set-default "$(git remote get-url "${REPOSITORY}")"
 
 echo "Creating pull request to publish release documentation to website."
 # Open PR to merge branch to `main` in `kroxylicious/kroxylicious.github.io`
-BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website"
+BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website.\n\nRemember to replace the container image SHAs in _date/release/[this-version].yaml, when they are available, before merging!"
```

Comment:
We could template in the exact path to update, but we'd have to have to convert dots to underscores and keep it in sync